dts/bindings/dma/nxp,mcux-edma.yaml

```diff
@@ -65,6 +65,18 @@ properties:
     description: |
       eDMA IP revision number.
 
+  channels-shared-irq-mask:
+    type: array
+    description: |
+      Describes channel enabled mask value on every IRQ.
+      The channel number is mapped to the bit value of array element value.
+      If the interrupt is shared on one channel number, the correspongding
+      bit is set to 1.
+      Please note each element of the array must be 32-bit. If there are more
```

Comment:
Might be worth specifying that the elements should be contiguous for the case where more than 32 channels exist

Reply:
Updated.